help/view-someones-profile.md

```diff
@@ -58,6 +58,19 @@ Additional tabs showing:
 
 {end_tabs}
 
+## Share someone's profile
+
+{start_tabs}
+
+{tab|desktop-web}
+
+{!right-sidebar-view-profile.md!}
+
+1. Click on the **link** (<i class="zulip-icon zulip-icon-link"></i> )
+   to the right of their name to copy the link to their profile.
```

Comment:
`tab|mobile` is missing. In mobile app, we can share if we click and hold the user, and then click on share in the popup menu.

Reply:
thanks I was not aware of that fact.

Reply:
There's no way to currently get the URL for a user profile in the mobile app currently.